src/builder/V2DutchOrderBuilder.ts

```diff
@@ -110,7 +103,7 @@ export class V2DutchOrderBuilder extends OrderBuilder {
       output.startAmount.gte(output.endAmount),
       `startAmount must be greater than endAmount: ${output.startAmount.toString()}`
     );
-    this.info.outputs.push(output);
+    this.info.outputs?.push(output);
```

Comment:
shouldn't need ? here if initialized in constructor? unless diff types in inherited classes from UnsignedXV2Order

Reply:
fixed in #119